app/metrics.yaml

```diff
@@ -539,3 +539,21 @@ custom_tab:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+
+preferences:
+  preference_toggled:
+    type: event
+    description: >
+      A user toggled a preference switch in settings
+    extra_keys:
+      preferenceKey:
+        description: "The preference key for the switch preference the user toggled"
```

Comment:
Yeah, I was thinking about if this needs all possible values - it would be helpful for whomever is looking at this from a data perspective to know what the values are (on the other hand, these are values that we're including that might just get out of date). Slight preference for adding those values though!